apisix/plugins/limit-count.lua

```diff
@@ -139,7 +139,7 @@ local _M = {
 
 
 local function group_conf(conf)
-    return conf
+    return core.table.deepcopy(conf)
```

Comment:
What about maintaining a deep copy of schema, so we only need to deep copy once?
https://github.com/apache/apisix/blob/a8a692d9a58a9a0185e22cfebbe2c0f000ba41f3/apisix/plugins/limit-count.lua#L86